dozer/cogs/levels.py

```diff
@@ -284,86 +302,129 @@ async def setrolelevel(self, ctx, role: discord.Role, level: int):
+            enabled = "Enabled" if ent.enabled else "Disabled"
+            embed.add_field(name=f"Levels are {enabled} for {ctx.guild}", value=f"XP min: {ent.xp_min}\n"
+                                                                                f"XP max: {ent.xp_max}\n"
+                                                                                f"Cooldown: {ent.xp_cooldown} Seconds\n"
+                                                                                f"Notification channel: {lvl_up_msgs}")
             await ctx.send(embed=embed)
 
-    configureranks.example_usage = """
-        `{prefix}configureranks --min 5 --max 15 --cooldown 15:` To configure rankings with a min value of 5, max value of 15 and a cooldown of 15 seconds
-        `{prefix}configureranks --disable:` To disable rankings
-        """
-
-    @command()
+    @command(aliases=["ranksettings", "levelsettings"])
     @guild_only()
     async def getlevelsconfig(self, ctx):
         """Returns the level settings for the current guild"""
-        settings = self._guild_settings[ctx.guild.id]
-
-        embed = discord.Embed(color=blurple)
-        embed.set_footer(text='Triggered by ' + ctx.author.display_name)
-
-        notify_channel = ctx.guild.get_channel(settings.lvl_up_msgs)
+        settings = self._guild_settings.get(ctx.guild.id)
+        if settings:
+            embed = discord.Embed(color=blurple)
+            embed.set_footer(text='Triggered by ' + ctx.author.display_name)
 
-        enabled = "Enabled" if settings.enabled else "Disabled"
-        embed.set_author(name=ctx.guild, icon_url=ctx.guild.icon_url)
-        embed.add_field(name=f"Levels are {enabled} for {ctx.guild}", value=f"XP min: {settings.xp_min}\n"
-                                                                            f"XP max: {settings.xp_max}\n"
-                                                                            f"Cooldown: {settings.xp_cooldown} Seconds\n"
-                                                                            f"Notification channel: {notify_channel}")
+            notify_channel = ctx.guild.get_channel(settings.lvl_up_msgs)
 
-        await ctx.send(embed=embed)
+            enabled = "Enabled" if settings.enabled else "Disabled"
+            embed.set_author(name=ctx.guild, icon_url=ctx.guild.icon_url)
+            embed.add_field(name=f"Levels are {enabled} for {ctx.guild}", value=f"XP min: {settings.xp_min}\n"
+                                                                                f"XP max: {settings.xp_max}\n"
+                                                                                f"Cooldown: {settings.xp_cooldown} Seconds\n"
+                                                                                f"Notification channel: {notify_channel}")
+            await ctx.send(embed=embed)
+        else:
+            await ctx.send("Levels not configured for this server")
```

Comment:
This should be merged into the `configureranks` command. Settings commands usually serve to both get and set settings, for example the `&starboard` command.

Reply:
Hmm, I don't think this got resolved. Instead of telling the user an invalid command was called, `&configureranks` should show the current configuration, instead of having a separate `&getlevelsconfig` command.